src/java/com/sarality/app/view/dialog/CustomViewDialogFragment.java

```diff
@@ -30,6 +32,12 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
   }
 
   @Override
+  public void onStart() {
```

Comment:
actually it is fine in render. i just tried it again. createDialog was too early.